server/webhook/merge_request.go

```diff
@@ -33,7 +38,18 @@ func (w *webhook) handleDMMergeRequest(event *gitlab.MergeEvent) ([]*HandleWebho
 		case actionReopen:
 			message = fmt.Sprintf("[%s](%s) reopen your merge request [%s!%v](%s)", senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername), event.ObjectAttributes.Target.PathWithNamespace, event.ObjectAttributes.IID, event.ObjectAttributes.URL)
 		case actionUpdate:
-			// TODO not enough check (opened/update) to say assigned to you...
+			toUsers = []string{}
+			for _, currentAssigneeID := range event.ObjectAttributes.AssigneeIDs {
+				assignedInPrevious := false
+				for _, previousAssignee := range event.Changes.Assignees.Previous {
+					if previousAssignee.ID == currentAssigneeID {
+						assignedInPrevious = true
```

Comment:
If `assignedInPrevious` is True, we could `break` this loop so it doesn't continue looping because it's not necessary.

Reply:
ah, good catch. thanks